website/meta/universe.json

```diff
@@ -327,15 +394,20 @@
             "pip": "spaczz",
             "code_example": [
                 "import spacy",
-                "from spaczz.pipeline import SpaczzRuler",
+                "from spaczz.matcher import FuzzyMatcher",
                 "",
-                "nlp = spacy.blank('en')",
-                "ruler = SpaczzRuler(nlp)",
-                "ruler.add_patterns([{'label': 'PERSON', 'pattern': 'Bill Gates', 'type': 'fuzzy'}])",
-                "nlp.add_pipe(ruler)",
+                "nlp = spacy.blank(\"en\")",
+                "text = \"\"\"Grint Anderson created spaczz in his home at 555 Fake St,",
+                "Apt 5 in Nashv1le, TN 55555-1234 in the US.\"\"\"  # Spelling errors intentional.",
```

Comment:
`\"` within the JSON string should render fine as `"` – I agree it makes the examples a bit less readable in the JSON but if we used single quotes here, it wouldn't match our normal code style for Python that we use in the docs so that would be less nice. So I'd vote for keeping it this way.

Reply:
You're right, but I guess the added confusion is having a multi-line string here, not sure that's actually necessary (cf the original example)

Reply:
The problem i saw is that there are no linebreaks in the code-example-blocks, so you are forced to scroll to the far right to read the whole text. An extrem example would be `pytextrank` where i also did the multi-line format.

Reply:
Ok, sure!